4.2.1 release (#80)
blackbaud/skyux-colorpicker: CHANGELOG.md

# 4.2.1 (2020-12-02)

> - Fixed the colorpicker component button to show proper hover and focus styles. [#77](https://github.com/blackbaud/skyux-colorpicker/pull/77)

Reviewer: ```suggestion
- Fixed the hover and focus styles for the colorpicker button. [#77](https://github.com/blackbaud/skyux-colorpicker/pull/77)
```

> - Fixed the colorpicker component to properly position the picker below the trigger button. [#79](https://github.com/blackbaud/skyux-colorpicker/pull/79)

Reviewer: ```suggestion
- Fixed the colorpicker component to position the picker below the trigger button. [#79](https://github.com/blackbaud/skyux-colorpicker/pull/79)
```
